metron-platform/metron-parsers/src/main/config/zookeeper/parsers/asa.json

```diff
@@ -0,0 +1,7 @@
+{
+  "parserClassName": "org.apache.metron.parsers.asa.BasicAsaParser",
+  "sensorTopic": "asa",
+  "parserConfig": {
+    "deviceTimeZone": "UTC-05:00"
```

Comment:
This is nice. If I have different devices in different time zones, I can just land each one in a different Kafka topic and configure the appropriate device time zone per topic.